v2.0/releases/index.md

```diff
@@ -6,6 +6,40 @@ The following table shows component versioning for Calico  **{{ page.version }}*
 
 Use the version selector at the top-right of this page to view a different release.
 
+
+## v2.0.1 - Release Notes
+
+This is a bug fix release, the major fixes and enhancements include:
+- Fix: some policies not working because of / in the labels.
+- Fix that ipv4_nat was not working (for OpenStack floating IPs)
+- Fix logic in static link test. Move inside container.
+- Fix configure_ipip_device() to pass IPAddress to set_interface_ips().
```

Comment:
You can see the Felix release notes at https://github.com/projectcalico/felix/releases.  They're different from raw commit messages - and I hope better - because the Felix process explicitly includes a review and editing step here.  So I recommend you use the release notes as input for what you write here, instead of going back to the Felix commit messages.